api/app/src/Entity/CourtOrderDeputy.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace App\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * Link table between court orders and deputies.
+ *
+ * @ORM\Table(name="court_order_deputy")
+ *
+ * @ORM\Entity()
+ *
+ * @ORM\HasLifecycleCallbacks()
+ */
+class CourtOrderDeputy
+{
+    /**
+     * @ORM\Id
+     *
+     * @ORM\ManyToOne(targetEntity="App\Entity\CourtOrder", inversedBy="courtOrderDeputyRelationships", cascade={"persist"})
+     *
+     * @ORM\JoinColumn(name="court_order_id", referencedColumnName="id")
+     */
+    private CourtOrder $courtOrder;
+
+    /**
+     * @ORM\Id
+     *
+     * @ORM\ManyToOne(targetEntity="App\Entity\Deputy", inversedBy="courtOrderDeputyRelationships", cascade={"persist"})
+     *
+     * @ORM\JoinColumn(name="deputy_id", referencedColumnName="id")
+     */
+    private Deputy $deputy;
+
+    /**
+     * @ORM\Column(name="discharged", type="boolean", nullable=false)
+     */
+    private bool $discharged;
+
+    public function getDeputy(): Deputy
```

Comment:
is there a way to set up false as the default for this? maybe in constructor or here. Not sure where :)